utils/email-helper.js

```diff
@@ -0,0 +1,60 @@
+const config = require('config');
+const sgMail = require('@sendgrid/mail');
+
+const recipient = config.get('recipient.to');
+const sender = config.get('sender.from');
+
+const sendEmail = async (message) => {
```

Comment:
This name `message` might look like a string type. (Because people use this name for string type variable often)

I think it can give some misleading idea. How about calling it as `params`, `emailParams` or `messageObj`?